src/libs/antares/study/include/antares/study/study.hxx

```diff
@@ -30,33 +30,35 @@ inline bool Study::readonly() const
     return (parameters.readonly);
 }
 
-template<unsigned int TimeSeriesT>
-inline void Study::storeTimeSeriesNumbers(Solver::IResultWriter& resultWriter) const
-{
-    storeTimeseriesNumbers<TimeSeriesT>(resultWriter, areas);
-}
+// {
+//     switch (TS)
+//     {
+//     case TimeSeriesType::timeSeriesShortTermInflows:
+//         // TODO
+//         destroyAllHydroTSGeneratorData();
+//         break;
+//     case TimeSeriesType::timeSeriesShortTermAdditionalConstraints:
+//         // TODO
+//         destroyAllHydroTSGeneratorData();
+//         break;
+//     default:
+//         break;
+//     }
+// }
 
-template<enum TimeSeriesType TS>
-inline void Study::destroyTSGeneratorData()
-{
-    switch (TS)
-    {
-    case TimeSeriesType::timeSeriesLoad:
-        destroyAllLoadTSGeneratorData();
-        break;
-    case TimeSeriesType::timeSeriesSolar:
-        destroyAllSolarTSGeneratorData();
-        break;
-    case TimeSeriesType::timeSeriesWind:
-        destroyAllWindTSGeneratorData();
-        break;
-    case TimeSeriesType::timeSeriesHydro:
-        destroyAllHydroTSGeneratorData();
-        break;
-    default:
-        break;
-    }
-}
+//
+// template<>
+// void Study::destroyTSGeneratorData<TimeSeriesType::timeSeriesShortTermInflows>()
+// {
+//     areas.each(
+//       [](Data::Area& area)
+//       {
+//           for (auto& sts: area.shortTermStorage.storagesByIndex)
+//           {
+//               sts.series->inflowsTSNumbers.reset();
+//           }
+//       });
+// }
```

Comment:
We don't do TS generation for short-term storage, you can remove this commented code